internal/v1/handler_blueprints_test.go

```diff
@@ -1639,16 +1639,33 @@ func TestFixupBlueprint(t *testing.T) {
 		PolicyId: uuid.MustParse(mocks.PolicyID2),
 	}))
 
+	var uo v1.UploadRequest_Options
+	require.NoError(t, uo.FromAWSS3UploadRequestOptions(v1.AWSS3UploadRequestOptions{}))
+
 	cases := []struct {
-		blueprint  v1.BlueprintBody
-		lintErrors []v1.BlueprintLintItem
+		blueprint   v1.BlueprintBody
```

Comment:
Just change the type from `v1.BlueprintBody` to `v1.CreateBlueprintRequest`, then there's no need to define name and description separately.